### PR TITLE
add codecov example config, use default homedir for config & secrets

### PR DESCRIPTION
**Pull request by oraclown**
Adds an example codecov badge and updates the github action to use codecov. Will need to update to the correct codecov badge, rather than my forked repo's one. [Relevant issue](https://github.com/tellor-io/pytelliot/issues/14)

Updates the playground contract address.

Uses the `default_homedir()` utility to retrieve secrets for the reporter.

Uses [eth gas station](https://ethgasstation.info/) instead of hard-coded gas price of 3 gwei.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![codecov](https://codecov.io/gh/oraclown-test/pytelliot/branch/main/graph/badge.svg?token=jDGxaDXc9Q)](https://codecov.io/gh/oraclown-test/pytelliot)
```
Comment (clamdad) [inline README.md:1]: Should this be served from tellor cocecod acct?

**Review by tallywiesenberg — APPROVED** [review]
lgtm. when i submit my pr we can update the web3 connection to go through the RPCEndpoint class

File: README.md (hunk 1 above)
Comment (oraclown) [reply README.md]: done

**Comment by oraclown** [comment]
> lgtm. when i submit my pr we can update the web3 connection to go through the RPCEndpoint class

sounds good

**Pull request merged**